src/data/faq_de.json

```
@@ -1160,7 +1159,7 @@
                                     "<li><b>Das Testergebnis auf anderem Weg erhalten</b></li>",
                                     "Wenn Sie das Testergebnis nicht über die Corona-Warn-App erhalten haben, kontaktieren Sie bitte Ihr zuständiges ärztliches Fachpersonal, Testzentrum, bzw. Gesundheitsamt. Auch wenn Ihr positives Testergebnis nicht in der Corona-Warn-App vorliegen sollte, Ihnen aber ein positives Testergebnis außerhalb der Corona-Warn-App mitgeteilt wurde, können Sie andere Personen über die App warnen, siehe: <a href='#qr_test' target='_blank'>Ich habe einen PCR-Test-QR-Code eingescannt, aber das Testergebnis konnte über Tage nicht abgerufen werden.</a></ul>",
                                     "<b>Können mehrere Tests registriert werden?</b>",
-                                    "Sie können je einen persönlichen PCR- und einen Schnelltest einscannen. Jeder QR-Code für einen Test kann nur auf einem Gerät eingescannt werden. Wenn das Testergebnis anschließend gelöscht wurde, kann es zwar für 30 Tage aus dem Papierkorb wiederhergestellt werden, jedoch nicht auf einem anderen Gerät gescannt werden. Dies dient dazu, Missbrauch und Falschmeldungen zu vermeiden. Weiter Informationen finden Sie hier: <a href='#QRcodes' target='_blank'>Was mache ich, wenn ich mehrere PCR-Test-QR-Codes habe?</a>",
+                                    "Sie können pro Familienmitglied je einen persönlichen PCR- und einen Schnelltest einscannen. Wählen Sie hierfür nach dem Scan des Tests aus, ob dieser für Sie selbst, oder für ein Familienmitglied bestimmt ist. Der Status eines Tests, unabhängig davon ob er zugehörig zu Ihnen selbst oder einem Familienmitgliedi ist, lässt sich im Anschluss auf der Status-Registerkarte abrufen. Jeder QR-Code für einen Test kann nur auf einem Gerät eingescannt werden. Wenn das Testergebnis anschließend gelöscht wurde, kann es zwar für 30 Tage aus dem Papierkorb wiederhergestellt werden, jedoch nicht auf einem anderen Gerät gescannt werden. Dies dient dazu, Missbrauch und Falschmeldungen zu vermeiden. Weiter Informationen finden Sie hier: <a href='#QRcodes' target='_blank'>Was mache ich, wenn ich mehrere PCR-Test-QR-Codes habe?</a>",

```

Remove "persönlichen". 
```suggestion
                                    "Sie können pro Familienmitglied je einen PCR- und einen Schnelltest einscannen. Wählen Sie hierfür nach dem Scan des Tests aus, ob dieser für Sie selbst, oder für ein Familienmitglied bestimmt ist. Der Status eines Tests, unabhängig davon ob er zugehörig zu Ihnen selbst oder einem Familienmitgliedi ist, lässt sich im Anschluss auf der Status-Registerkarte abrufen. Jeder QR-Code für einen Test kann nur auf einem Gerät eingescannt werden. Wenn das Testergebnis anschließend gelöscht wurde, kann es zwar für 30 Tage aus dem Papierkorb wiederhergestellt werden, jedoch nicht auf einem anderen Gerät gescannt werden. Dies dient dazu, Missbrauch und Falschmeldungen zu vermeiden. Weiter Informationen finden Sie hier: <a href='#QRcodes' target='_blank'>Was mache ich, wenn ich mehrere PCR-Test-QR-Codes habe?</a>",

```